core/src/main/java/bio/terra/pearl/core/dao/study/StudyDao.java

```diff
@@ -84,9 +84,9 @@ public Optional<Study> findOneFullLoad(UUID id) {
      * So, for example, if a portal has two studies, this might return the 'sandbox' environment for
      * both studies
      */
-    public List<Study> findWithPreregContent(String portalShortcode, EnvironmentName envName) {
+    public List<Study> findWithPreEnrollContent(String portalShortcode, EnvironmentName envName) {
```

Comment:
Pre-reg is portal-scoped, pre-enroll is study scoped, right? So this was just renaming it to be correct?

(Just making sure I've got the terminology straight in my head)

Reply:
yes, this is renaming to be correct, not changing behavior